apps/classroomio-com/src/app.html

```diff
@@ -98,6 +98,11 @@
       layout: 'month_view'
     });
   </script>
+
+  <script type="text/javascript">
+    window.localStorage.setItem('umami.disabled', 1)
+  </script>
```

Comment:
@codiini Shouldn't we be doing this only if the environment is localhost?

Won't this disable umami completely?

Reply:
From the docs here, it completely stops umami from collecting analytics https://umami.is/docs/guides/exclude-my-own-visits

I think what we should be doing is updating `src/routes/+layout.svelte`

```ts
import { dev } from '$app/environment';
import { onMount } from 'svelte';

onMount(() => {
  if (dev) {
    localStorage.setItem('umami.disabled', 1)
  }
})
```

Reply:
Oh yes, my bad. I ready the linked Discussion on the PR and skipped straight to the solution without looking at further context. Fixing now.